pynest/nest/lib/hl_api_info.py

```diff
@@ -55,26 +58,37 @@ def authors():
 
 
 @check_stack
-def helpdesk(browser="firefox"):
-    """Open the NEST helpdesk in the given browser.
+def helpdesk():
+    """Open the NEST helpdesk in browser.
 
-    The default browser is firefox.
-
-    Parameters
-    ----------
-    browser : str, optional
-        Name of the browser to use
+    Use the system default browser.
     """
+    if 'NEST_DOC_DIR' not in os.environ:
+        print(
+            'NEST help needs to know where NEST is installed.'
+            'Please source nest_vars.sh or define NEST_DOC_DIR manually.')
+        return
 
-    sr("/helpdesk << /command (%s) >> SetOptions" % browser)
-    sr("helpdesk")
+    url = os.path.join(os.environ['NEST_DOC_DIR'] + "/help", "helpindex.html")
```

Comment:
I still have some issues with this line:
1. This is not a URL, but the name of a local file to be opened by a browser, so the variable name should be `helpfile` or similar, not `url`.
2. Instead of mixing string concatenation and `os.path.join()`, just use `os.path.join()`:
```python
helpfile = os.path.join(os.environ['NEST_DOC_DIR'], 'help', 'helpindex.html')
```